README.md

```diff
@@ -1,62 +1,82 @@
   - Write a `systemd` definition that will start/stop your named container
   - Ensure `systemd` will restart your container/service if EC2 instance is rebooted
   - Ensure `systemd` will restart your container/service if it is terminated with  `docker kill`
-  - Once you’re satisfied, use Git to commit your `systemd` definition to your local Git branch
+  - Once you’re satisfied, use git to commit your `systemd` definition to your local git branch
 - Make dockerized web-service write logs to  `/var/log`
   - Using the Docker driver, volumes, a sidecar or a logging agent ensure that your container is logging to a file in `/var/log` on the EC2 host
+
+## Part 3
+_Goal: Containerize an your own hello-world app using Docker_ 
 - Create a `hello-world` style app and containerize it
   - Write a basic script in a language of your choice that echoes “Hello world” to stdout 
   - Write a new `Dockerfile` that will build your script into a Docker image and execute it when the container is run. Ensure any dependencies necessary are included
   - Modify your script to echo an environment variable fed in on the `docker` command line at runtime
   - Ensure that your application functions as expected with  `docker run` and a provided `-e` flag 
-  - Once you’re satisfied that your application and container function as intended, commit your code to the local Git branch
+  - Once you’re satisfied that your application and container function as intended, commit your code to the local git branch
+
+## Part 4
+_Goal: Install minikube and deploy an existing app into a Kubernetes Cluster_  
+What We're Looking For: 
 - Install and run `minikube` 
   - Following the official documentation, install the correct flavor of `minikube` on the EC2 instance
   - After your minikube cluster is started, ensure `kubectl` is communicating with it properly
 - Deploy an application onto minikube cluster
-  - In your home directory, clone the following Git repository : https://github.com/reactiveops/k8s-workshop
+  - In your home directory, clone the following git repository : https://github.com/reactiveops/k8s-workshop
   - Use the resources within the `common` directory to apply/deploy the workshop’s application and its dependencies to your minikube cluster
   - Ensure the application and its dependencies are deployed in the `k8s-workshop` namespace
   - Ensure that pods of each of the three Deployments within are running successfully
     - You may need to troubleshoot Pods and Deployments with the `kubectl describe`  or `kubectl logs` commands
     - Note any changes you may have made to ensure the applications and services function
-  - (Optional) Create a public git repository in  github/gitlab, push the local branch of the `tethys` repository with your changes there and share with us.
+  - (Optional) Create a public git repository in github/gitlab, push the local branch of the `tethys` repository with your changes there and share with us.
```

Comment:
This goes against what I was saying before about optional... 
I was thinking initially that we shouldn't require someone to create a new account somewhere if they don't already have one. That's probably silly though since nearly everyone will have a gitlab account... What do you think?

Reply:
Yeah, let's take off all the _optional_ tags.